libs/webb-ui-components/src/components/Chip/types.ts

```diff
@@ -25,12 +25,15 @@ export interface ChipProps extends WebbComponentBase {
    * If `true`, the chip will display as disabled state
    */
   isDisabled?: boolean;
+  isSelected?: boolean;
+  onClick?: React.EventHandler<React.MouseEvent | React.KeyboardEvent>;
```

Comment:
The `onClick` is already inside the `WebbComponentBase` interface. We don't need to redefine it here.